assets/src/dashboard/app/api/apiProvider.js

```diff
@@ -49,15 +49,26 @@ export default function ApiProvider({ children }) {
   const [stories, setStories] = useState([]);
 
   const fetchStories = useCallback(
-    async ({ status = STORY_STATUSES[0].value }) => {
+    async ({ status = STORY_STATUSES[0].value, searchTerm }) => {
+      const perPage = '100'; // TODO set up pagination
+      const query = searchTerm
+        ? {
+            status,
+            search: `${searchTerm}`,
```

Comment:
What's the intention behind the template literals here?

Also, could this be something like:

```js
const query {
  status,
  per_page: perPage,
  search: searchTerm ? searchTerm : undefined,
};
```

Untested, but looks more readable.

Reply:
Hey good question! So, I don't want search to show up in the query at all if searchTerm is an empty string (default value) and the template literal is because when i was testing it the search term wouldn't work if, say, there's a space in the search without it.  

If you can point me to more details about where this query lives on the php side of thing perhaps there's some encoding that could be done there.  

I'll play with something that could be more 'readable', honestly this is going to be iterative since we need pagination here as well as the ability to total stories and i was just trying to get to the point where it's usable for an alpha as quick as possible.

Reply:
took a second look at this and yeah, thanks for the call out. ended up with this for now: 
```
const query = {
        status,
        per_page: perPage,
        search: searchTerm || undefined,
      };
```

I swear the template literal was making a difference yesterday but now it's fine. Must have been some weird combo.